.github/workflows/check-pr.yaml

```diff
@@ -11,7 +11,13 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+    - name: Setup python 3.9
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
     - name: Install pre-commit
       run: pip install pre-commit
+    - name: Clean pre-commit dir
+      run: pre-commit clean
```

Comment:
are these 2 changes here needed b/c of the ansible-lint change?

Reply:
no, not really. I'll propose a different PR for those changes